applications/tari_validator_node/src/base_layer_scanner.rs

```diff
@@ -284,7 +284,23 @@ impl BaseLayerScanner {
                 })?;
                 match sidechain_feature {
                     SideChainFeature::ValidatorNodeRegistration(reg) => {
-                        self.register_validator_node_registration(current_height, reg).await?;
+                        let validator_node_timeout = self
+                            .base_node_client
+                            .get_consensus_constants(tip.height_of_longest_chain)
```

Comment:
Is this preferrable to use `end_height`, instead ?

Reply:
Currently the base layer uses the tip so let's use the tip for now

Reply:
Perfect, thanks for the hint !